libs/wingsdk/src/ex/react-website.md

```diff
@@ -0,0 +1,86 @@
+---
+title: React Website
+id: react-website
+description: A built-in resource for creating deployable websites using the React framework.
+keywords: [Website, React, deployment, build]
+sidebar_position: 1
+---
+
+The `ex.ReactWebsite` resource represents a website, built using React, that can be both hosted in the cloud or to run on a development hot-reload server in the sim target.
+
+## Usage
+
+### Initialization
+
+Using the default arguments:
+
+```ts
+bring cloud;
```

Comment:
Shouldn't we bring ex; instead of cloud here? The whole example should be for an `ex.ReactWebsite` instead of a cloud.Website or did I get it wrong?